pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/subjects/SierraOrganisationSubjectsTest.scala

```diff
@@ -228,20 +230,12 @@ class SierraOrganisationSubjectsTest
     // throw and have somebody inspect the record manually.  It's not at all
     // clear what we should do, as we don't have enough to populate the
     // Organisation label.
-    it("errors if there's nothing in subfield a or b") {
+    it("returns zero subjects if there's nothing in subfield a or b") {
```

Comment:
This is a substantive change to the behaviour.  I couldn't see a simple way to reconcile throwing here with returning Nil elsewhere, and I decided it wouldn't be wroth the effort.

Given that I've never seen this error, and our normal modus operandi is to warn-and-ignore, I thought I'd go with this approach.